<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInfoItem</class>
 <widget class="QWidget" name="VideoInfoItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QMyMoveLable" name="wd_movie" native="true">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <widget class="QLabel" name="lb_name">
         <property name="text">
          <string>名字</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_size">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>大小</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMyMoveLable</class>
   <extends>QWidget</extends>
   <header>qmymovelable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
